.github/workflows/_tauri.yml

```diff
@@ -36,7 +36,7 @@ jobs:
             syms-artifact: rust/gui-client/firezone-linux-gui-client-amd64.dwp
             pkg-artifact: rust/gui-client/firezone-linux-gui-client_amd64.deb
           - runs-on: windows-2019
-            binary-dest-path: firezone-windows-client
+            binary-dest-path: firezone-client-gui-windows
```

Comment:
Putting the OS name here is redundant since every package type (MSI, dmg, deb) belongs to exactly one OS, but I think it's easier to read "firezone-client-gui-windows" and stop there than to read "firezone-client-gui.......... msi"

Reply:
Makes sense